app/controllers/events_controller.rb

```diff
@@ -1,38 +1,14 @@
 class EventsController < ApplicationController
-  
   def index
     @events = Event.all
```

Comment:
```suggestion
    if param[:web_source].present?
        @events = Event.where("websource LIKE ?", params[:websource]
    else
    @events = Event.all
```

Reply:
yes added the functionality . now index controller is using params to filter events based on parameters .